docs/web_quickstart.rst

```diff
@@ -450,8 +450,14 @@ third-party library, :mod:`aiohttp_session`, that adds *session* support::
 
     async def handler(request):
         session = await get_session(request)
-        last_visit = session['last_visit'] if 'last_visit' in session else None
-        text = 'Last visited: {}'.format(last_visit)
+
+        if "last_visit" in session:
+            last_visit = session["last_visit"]
+        else:
+            last_visit = None
+            session["last_visit"] = time.time()
```

Comment:
Maybe this would be more concise?
```suggestion
        last_visit = session.get("last_visit")
        if last_visit is None:
            session["last_visit"] = time.time()
```

Reply:
Wait, don't we want to update last_visit everytime, otherwise it wouldn't be the _last_ visit...

```suggestion
        last_visit = session.get("last_visit")
        session["last_visit"] = time.time()
```